### ISSUE TITLE
Add JWT example with @Guard decorator

### ISSUE THREAD

**Issue by pie6k** [prismake/typegql#8]

**Comment by lorefnon** [comment]
I have been working on [an application](https://github.com/lorefnon/bugbox) for personal evalation. It is very early prototype but features client-server JWT auth. 

I plan to integrate hierarchical role based authorisation in near future.

**Comment by pie6k** [comment]
If you'd like to extract it from your app and create PR with example, that would be so awesome @lorefnon :)

**Comment by lorefnon** [comment]
Sure, will do. 

I just realized that `@Guard` no longer exists in master. I will update my example to use `@Before` and create a PR shortly.

**Comment by pie6k** [comment]
Right. I've removed it as `@Guard` was simply not needed abstraction. It was simple function in form of 

```
function Guard(options) {
  return Before(otherOptions)
}
```

as I think it's good approach to keep `graphql` more like library than framework, I've decided such `Guard` is bad abstraction.
